### DISCUSSION TITLE
Welcome to docker-rspamd Discussions!

### DISCUSSION THREAD

**Discussion by tiredofit** [tiredofit/docker-rspamd#5] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by expertaz-zz** [comment]
Hello,thank you for image.
Is this rspamd image include web interface?
Regards,
Hafiz.

**Reply by tiredofit** [reply]
Yes, it does include the web interface. It's listening at port 11334. It looks for the controller password upon visit.
